criu/cr-restore.c

```diff
@@ -3554,7 +3554,9 @@ static int sigreturn_restore(pid_t pid, struct task_restore_args *task_args, uns
 		CoreEntry *tcore;
 		struct rt_sigframe *sigframe;
 		k_rtsigset_t *blkset = NULL;
-
```

Comment:
The above blkset is an unused variable when CONFIG_MIPS is set.

Reply:
yes, I have ifdef it

Reply:
Thanks!